### PR TITLE
Update tests for shipping methods

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I added test for shipping zones bulk delete and updated tests for creating shipping zone and method

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://stable.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5502 Create missing tests for shipping methods](https://app.clickup.com/t/2549495/SALEOR-5502)

**Review by dominik-zeglen — APPROVED** [review]

File: cypress/integration/configuration/shippingMethods/postalCodes.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
 } from "../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Postal codes in shipping", () => {
+  describe("As a user I want to create shipping method with postal codes", () => {
```
Comment (eCzerniak) [inline cypress/integration/configuration/shippingMethods/postalCodes.js:26]: Those tests are failing. Probably some changes on UI were made. 
image

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Please check those tests.

**Review by eCzerniak — APPROVED** [review]

File: cypress/integration/configuration/shippingMethods/shippingWeights/shippingWeightsLimits.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ import {
 } from "../../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Shipping weight limits", () => {
```
Comment (Kjaroszew) [inline cypress/integration/configuration/shippingMethods/shippingWeights/shippingWeightsLimits.js:26]: This test fails during execution, probably changes to UI were made which resulted in error
image

File: cypress/integration/configuration/shippingMethods/shippingWeights/weightRecalculate.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ import filterTests from "../../../../support/filterTests";
 import { changeWeightUnit } from "../../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Recalculate weights", () => {
+  describe("As a staff user I want to change shop default weight unit", () => {
```
Comment (Kjaroszew) [inline cypress/integration/configuration/shippingMethods/shippingWeights/weightRecalculate.js:21]: This test fails during execution, probably changes to UI were made which resulted in error
image

File: cypress/integration/configuration/shippingMethods/channelsInShipping.js [hunk 4]
```diff
@@ -18,9 +16,13 @@ import * as shippingUtils from "../../../support/api/utils/shippingUtils";
...
 import { getFormattedCurrencyAmount } from "../../../support/formatData/formatCurrencyAmount";
 import { enterHomePageChangeChannelAndReturn } from "../../../support/pages/channelsPage";
+import {
+  enterAndSelectShippings,
+  enterShippingZone
+} from "../../../support/pages/shippingZones";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Channels in shippingMethod", () => {
+  describe("As a staff user I want have different shipping method prices for each channel", () => {
```
Comment (Kjaroszew) [inline cypress/integration/configuration/shippingMethods/channelsInShipping.js:25]: This test fails during execution, probably changes to UI were made which resulted in error
image

File: cypress/integration/configuration/shippingMethods/createShippingMethod.js [hunk 5]
```diff
@@ -19,15 +21,21 @@ import {
 } from "../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Create shipping method", () => {
+  describe("As a staff user I want to create shipping zone and rate", () => {
```
Comment (Kjaroszew) [inline cypress/integration/configuration/shippingMethods/createShippingMethod.js:24]: This test fails during execution, probably changes to UI were made which resulted in error
image

File: cypress/integration/configuration/shippingMethods/editShippingMethod.js [hunk 6]
```diff
@@ -19,7 +19,7 @@ import {
 } from "../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Edit shipping method", () => {
+  describe("As a user I should be able to update and delete shipping method", () => {
```
Comment (Kjaroszew) [inline cypress/integration/configuration/shippingMethods/editShippingMethod.js:22]: It seems that all found issues are similar in nature 
image

**Review by Kjaroszew — APPROVED** [review]
ok

**Pull request merged**